app/Http/Controllers/Api/ReportbackController.php

```diff
@@ -52,4 +54,15 @@ public function store(ReportbackRequest $request)
 
         return $this->item($reportback, $code);
     }
+
+    /**
+     * Update reportbackitem(s) that already exists.
+     *
+     * @param \Illuminate\Http\Request $request
+     * @return \Illuminate\Http\Response
+     */
+    public function updateReportbackItems(Request $request)
+    {
+        $this->reportbackService->updateReportbackItems($request->all());
```

Comment:
@sbsmith86 ah yes sorry - I changed everything in the Service and Repository as well to return the `$item` or `null` if there was an error! Just added this to the code in the Controller but still getting the same response as above? 
```
    public function updateReportbackItems(Request $request)
    {
        $item = $this->reportbackService->updateReportbackItems($request->all());

        if ($item === null ) {
            $code = 404;
        } else {
            $code = 201;
        }

        return $this->item($item, $code, $this->itemTransformer);
    }
```